src/main/java/uk/gov/hmcts/reform/cwrdapi/controllers/CaseWorkerRefController.java

```diff
@@ -132,4 +139,55 @@
                 .status(idamRolesMappingResponse.getStatusCode())
                 .body(idamRolesMappingResponse);
     }
+
+    @ApiOperation(
+            value = "This API gets the User details from Caseworker Profile",
+            authorizations = {
+                    @Authorization(value = "ServiceAuthorization"),
+                    @Authorization(value = "Authorization")
+            }
+    )
+    @ApiResponses({
+            @ApiResponse(
+                    code = 200,
+                    message = "Successfully fetched the Caseworker profile(s)",
+                    response = CaseWorkerProfile.class
+            ),
+            @ApiResponse(
+                    code = 400,
+                    message = BAD_REQUEST
+            ),
+            @ApiResponse(
+                    code = 401,
+                    message = UNAUTHORIZED_ERROR
+            ),
+            @ApiResponse(
+                    code = 403,
+                    message = FORBIDDEN_ERROR
+            ),
+            @ApiResponse(
+                    code = 404,
+                    message = NO_DATA_FOUND
+            ),
+            @ApiResponse(
+                    code = 500,
+                    message = INTERNAL_SERVER_ERROR
+            )
+    })
+    @PostMapping(
+            path = "/users/fetchUsersById/",
+            consumes = APPLICATION_JSON_VALUE,
+            produces = APPLICATION_JSON_VALUE
+    )
+    @Secured("cwd-admin")
+    public ResponseEntity<Object> fetchCaseworkersById(@RequestBody List<String> caseWorkerIds) {
+        long currentTimeMillis = System.currentTimeMillis();
+        if (CollectionUtils.isEmpty(caseWorkerIds)) {
+            throw new InvalidRequestException("Caseworker request is empty");
+        }
+        ResponseEntity<Object> response = caseWorkerService.fetchCaseworkersById(caseWorkerIds);
+        log.info("Time taken by fetchCaseworkersById method in Controller = "
```

Comment:
Is it really required to log the duration ?

Reply:
Removing the logging